Automattic/wp-calypso
# Block Editor: Make sure unsaved changes popup appears in multi entity editing context

#### Changes proposed in this Pull Request

* Add a confirmation popup about unsaved changes to Back to dashboard button

#### Testing instructions

* Open site editor
* Make some changes
* Open navigation sidebar
* Click back until Back to dashboard available
* Click on Back to dashboard and make sure a confirmation alert pops up



Related to #
